src/main/java/controllers/MemberController.java

```diff
@@ -0,0 +1,41 @@
+
+package controllers;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.ModelAndView;
+
+import services.BrotherhoodService;
+import domain.Brotherhood;
+
+@Controller
+@RequestMapping("/brotherhood/member")
+public class MemberController extends AbstractController {
+
+	@Autowired
+	private BrotherhoodService	brotherhoodService;
+
+
+	public MemberController() {
```

Comment:
```suggestion
	public MemberController(BrotherhoodService	brotherhoodService) {
	  this.brotherhoodService = brotherhoodService;
	  }
```

Reply:
You should consider inject service inside constructors to make unit tests easier. =)